INSTALL.md

```diff
@@ -27,6 +27,7 @@ https://downloads.sourceforge.net/project/ibmswtpm2/ibmtpm532.tar
 attention described below.
 
 # System User & Group
+`tpm2-abrmd` must run as user `tss` or `root`.
```

Comment:
This isn't strictly true is it? I think I see what you're getting at: the default is the user 'tss' but you can really run it as any user so long as the dbus config is right. I think the text following this addition says exactly this?

Reply:
Yes it is true, unless you hand-edit the undocumented tpm2-abrmd.conf and add some XML to allow another user.
Or are you referring to "sudo"? Which does change the user to root.

I can add text about hand-editing tpm2-abrmd.conf as an alternative.

Reply:
> This isn't strictly true is it? I think I see what you're getting at: the default is the user 'tss' but you can really run it as any user so long as the dbus config is right. I think the text following this addition says exactly this?

This is only partially true though, because is not only the access to the system bus, the daemon should also have access to the TPM2 device. And currently the user and group are hardcoded to `tss`.

Reply:
So this is completely true, because of the daemon and the /dev/tpm0 device.  I filed a separate bug to add documentation on adding users for abrmd and perhaps (I don't recommend) add a configure flag to change the default user from tss to some other user:
#349 Document how to add to permitted abrmd users and maybe how to change default user, tss